rq/queue.py

```diff
@@ -26,6 +26,7 @@
 from .serializers import resolve_serializer
 from .types import FunctionReferenceType, JobDependencyType
 from .utils import as_text, backend_class, compact, get_version, import_attribute, parse_timeout, utcnow
+from .dependency import Dependency
```

Comment:
Imports need to be sorted alphabetically.

Reply:
Resolved